facia-tool/public/js/utils/sparklines.js

```diff
@@ -140,7 +140,10 @@ function reduceRequest (memo, front, articles, options) {
 
         return memo;
     })
-    .catch(function () {});
+    .catch(function () {
+        // Ignore errors from Ophan
+        return {};
```

Comment:
The complete code is more like this

```
Promise.reject(new Error('whops')).catch(function () {}).then(function (data) {data.something})
```

The last `then` is used by consumers of that function. Because there are many consumers I prefer to guard upstream instead of doing `.then(function (data = {}) {data.something})` in several other places